mlflow/gateway/utils.py

```diff
@@ -68,7 +68,18 @@ def _is_valid_uri(uri: str):
         return False
 
 
-@experimental
+def gateway_deprecated(f):
+    deco = deprecated(
+        "2.9.0",
+        impact=(
+            "It will be replaced by the deployments API in a future release. "
+            "See https://mlflow.org/docs/latest/llms/gateway/deprecation.html for more details."
+        ),
+    )
+    return deco(f)
```

Comment:
The error message looks like this, which sounds a bit strange:

```
``mlflow.gateway.client.MlflowGatewayClient`` is deprecated. It will be replaced by the deployments API in a future release. See https://mlflow.org/docs/latest/llms/gateway/deprecation.html for more details. Use ``2.9.0`` instead.
```